Improved Pokemon List Transfer/PowerUp

nxy opened the pull request:
Removed execute() from Transfer/PowerUp. Added a fix to update Candy Tips after Transfer/PowerUp.

nxy commented:
![consoletoggle](https://cloud.githubusercontent.com/assets/1384861/17313777/ada5a31a-582c-11e6-84cf-4c67eac42910.png)
Toggle Show/Hide

nxy commented:
Hey Nova, do you want me to fix the conflicts or are you doing your own thing?

Novalys commented:
If you could do it that would be great, I'm tied to my day time job until 5
PM MST :c

On Aug 2, 2016 12:04 PM, "nxy" <notifications@github.com> wrote:

> Hey Nova, do you want me to fix the conflicts or are you doing your own
> thing?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Novalys/PokemonGo-Bot-SimpleGUI/pull/214#issuecomment-236989817>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAWg1rXZ0_YHM5IypFigVdjJQdEfzBMDks5qb4augaJpZM4JZ5X8>
> .
>

nxy commented:
Ok sure np

nxy commented:
@Novalys Should be all good now :+1:

nxy commented:
Ahh there we go.. Just getting used to Github

nxy commented:
![untitled-2](https://cloud.githubusercontent.com/assets/1384861/17442504/17c8c2d4-5b04-11e6-95df-521c999b8dd5.png)

nxy commented:
To summarize what I've done so far for the Pokemon List:

- Bug fixes
- Improved Transfer / Powerup / Sort functions
- Added Sort for Index Number / Newest
- Added the Search bar, allowing you to search by Name, Index Number, Type, and Nickname
- Pokemon images are now downloaded to the application folder for quick loading

- Added Console to the MainWindow with Toggle Show/Hide

@Novalys I'll be gone for a few days and won't be able to update my Pull Request when you update the api. Just wanted to let you know.

~nxy
